### PR TITLE
change chart library

### PR DESCRIPTION
**Pull request by esoliani**
using https://github.com/codesuki/react-d3-components, which has more options and charts.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -85,7 +85,7 @@
     "lodash": "3.10.1",
     "object-assign": "4.0.1",
     "react": "0.13.3",
-    "react-d3": "0.4.0",
+    "react-d3-components": "^0.5.2",
```
Comment (uniphil) [inline package.json]: remove the semver range `^`

**Comment by uniphil** [comment]
Awesome how little change it is to change out these libraries!

File: app/scripts/components/dashboard/charts/waterpoint-status-chart.jsx [hunk 2]
```diff
@@ -48,13 +48,12 @@ const WaterpointStatusChart = React.createClass({
...
               data={this.parseData(dataRes)}
-              fill="#3182bd"
               height={200}
               legend={true}
-              stackOffset="wigget"
+              margin={{top: 10, bottom: 50, left: 50, right: 10}}
               title={{k: 'chart.status-waterpoints.title'}}
               width={500}
-              xAxisLabel={{k: 'chart.status-waterpoints.x-axis'}}
+              xAxis={{innerTickSize: 6, label: {k: 'chart.status-waterpoints.x-axis'}}}
```
Comment (esoliani) [inline app/scripts/components/dashboard/charts/waterpoint-status-chart.jsx:56]: This does not work, so we need to improve TSetChildProp

Comment (uniphil) [inline app/scripts/components/dashboard/charts/waterpoint-status-chart.jsx:56]: tricky, I'll think about it

**Pull request merged**